src/services/pacs_integration.tsx

```diff
@@ -0,0 +1,111 @@
+import { DcmImage, PACSMainResponse } from "../context/reducers/dicomImagesReducer";
+import axios from 'axios';
+
+declare var process: {
+    env: {
+        REACT_APP_CHRIS_UI_PFDCM_URL: string,
+    }
+};
+
+class PACSIntegration {
+
+    private static basePACSFilePath = 'SERVICES/PACS/covidnet/';
```

Comment:
What do you think about keeping constant variables in a separate file so its easier to locate all constants and know where to modify them?

Reply:
Should we keep this value in the `.env` files? Or should we create a new JSON file for variables?

Reply:
Base file path is now stored in `app.config.tsx`

Reply:
Perfect